deployment/local-k8s/example-runner/stopK8sServices.sh

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env bash
+# Copyright 2018-2021 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+NAMESPACE=$1
+
+cd deployment
+
+if [ -z "$NAMESPACE" ]
+then
+    echo "Delete all the resources in the default namespace"
+    # Delete the existing helm deployment
+    helm delete palisade
+
+    # Delete all the resources
+    kubectl delete jobs --all
+    kubectl delete pods --all
```

Comment:
Same as mentioned previous, but you don't need to delete these at all. They'll be cleaned up by the helm delete
```suggestion
```

Reply:
The deployment job doesn't get deleted when doing the helm delete so we still need the job deletion command. It also takes about 60 seconds for the `helm delete palisade` to completely delete everything it can (excluding the pvs, pvcs and the deployment job)

Reply:
This would require a sleep to be put in place as the `helm delete` command doesn't continue while everything is deleted from Kubernetes.

Reply:
Updated this script after doing some local testing.